ios/Mattermost/Info.plist

```diff
@@ -64,25 +64,25 @@
 	<key>NSBluetoothPeripheralUsageDescription</key>
 	<string>Share post data across devices with Mattermost</string>
 	<key>NSCalendarsUsageDescription</key>
-	<string>Share contacts in your Mattermost instance</string>
+	<string>Share contacts in your Mattermost workspace</string>
 	<key>NSCameraUsageDescription</key>
-	<string>Take a Photo or Video and upload it to your Mattermost instance</string>
+	<string>Take a Photo or Video and upload it to your Mattermost workspace</string>
 	<key>NSFaceIDUsageDescription</key>
 	<string>Mattermost uses Face ID to restrict unauthorized users from accessing the app.</string>
 	<key>NSLocationAlwaysUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>Send your current location to your Mattermost instance</string>
+	<string>Send your current location to your Mattermost workspace</string>
```

Comment:
```suggestion
	<string>Send your current location to Mattermost</string>
```

Reply:
Same question as above. Also - why are there 2 strings that appear to be communicating the same message?

Reply:
you have 2 modes for the location permission.  One is allowing the MM app to know the user's location only when the user is using the app and the other is always granted irrespective if the app is in use.  This is as per iOS protocol.